models/comments_models.js

```diff
@@ -18,4 +18,25 @@ exports.postComment = (article_id, newComment) => {
         .query("INSERT INTO comments (author, body, article_id, votes) VALUES ($1, $2, $3, $4) RETURNING *;", [username, body, article_id, 0])
     })
     .then(({ rows }) => rows[0])
+}
+
+exports.checkCommentID = (comment_id) => {
+    return db
+    .query("SELECT * FROM comments WHERE comment_id = $1", [comment_id])
+    .then(({ rows }) => {
+        if(rows.length === 0){
+            return Promise.reject({status:404, msg: 'ID not found'})
+        }
```

Comment:
Great custom error, but I'd tweak this error message bit - the comment wasn't found, not the ID